sources/Tests/tests/ECS/ecs.cpp

```diff
@@ -228,3 +229,135 @@ TEST_CASE("game_events_handling", "[ecs]")
   gameWorld->emitEvent(TestEvent{20});
   REQUIRE(listener->getLastMessageCode() == 10);
 }
+
+TEST_CASE("game_objects_transform_components_management", "[ecs]")
+{
+    std::shared_ptr<GameWorld> gameWorld = GameWorld::createInstance();
+
+    std::vector<GameObject> gameObjects;
+    for(int i = 0; i < 100; i++) {
```

Comment:
Here and below it will be better to use type `size_t` for indices and counters. `size_t` always has enough size to store any index or object size. It is not guaranteed for int.

```
for (size_t i = 0; i < 100; i++) {
// some code
}
```